devtools/cli/src/main/java/io/quarkus/cli/BuildToolDelegatingCommand.java

```diff
@@ -123,7 +123,8 @@ public void prepare(BuildToolContext context) {
 
     public void prepareMaven(BuildToolContext context) {
         BuildSystemRunner runner = getRunner(context);
-        BuildSystemRunner.BuildCommandArgs compileArgs = runner.prepareAction("compiler:compile", context.getBuildOptions(),
+        BuildSystemRunner.BuildCommandArgs compileArgs = runner.prepareAction("resources:resources",
+                context.getBuildOptions(),
```

Comment:
@ia3andy @aloubyansky I'm not sure if this class is related to the plugins only or not. If not, I'd like to have your opinion on this change.